app.config.js

```diff
@@ -28,6 +28,7 @@ const extraConfig = {
     rules: [
       {
         test: /\.worker(\.entry)\.js$/,
+        issuer: { not: [/node_modules\//] },
```

Comment:
We want to use our custom worker loader, unless the worker is loaded by a dependency, in which case we leave it alone.

Reply:
Can you add this comment in the file? Since it a JS file, let's enjoy being able to write comments :D